docs/features/specification.md

```diff
@@ -1,9 +1,4 @@
-# Overview
-
-The Rollout object has two available strategies: Canary and BlueGreen. Below are the links to the documenation for each strategy:
-
-1. [Blue Green](bluegreen.md)
-1. [Canary](canary.md)
+# Rollout Specification
```

Comment:
I'm wondering if we may need some automation to make sure this specification is up to date. For example, I noticed that the postPromotionAnalysis is missing from the blueGreen field.